UnicodeJsps/src/main/webapp/properties.jsp

```diff
@@ -20,7 +20,8 @@ th           { text-align: left }
         UtfParameters utfParameters = new UtfParameters(queryString);
 
         String propForValues = utfParameters.getParameter("a");
-		UnicodeJsp.showPropsTable(out, propForValues, "properties.jsp");
+        // TODO(egg): Cache this page.
```

Comment:
Ok for now. I think just a static page with an explicit list of the properties (and if enumerated, the values) without links would be very helpful for the users, and not hard to generate at build time.

Reply:
Yes. And we should probably grab some information from IndexUnicodeProperties as well, e.g., which files they come from.

But not today.

Reply:
Agreed